IdentityCore/src/controllers/MSIDSilentController.m

```diff
@@ -91,18 +93,23 @@ - (void)acquireToken:(nonnull MSIDRequestCompletionBlock)completionBlock
 
     [silentRequest executeRequestWithCompletion:^(MSIDTokenResult * _Nullable result, NSError * _Nullable error)
     {
-
+        MSIDRequestCompletionBlock completionBlockWrapper = ^(MSIDTokenResult * _Nullable result, NSError * _Nullable error)
```

Comment:
Same here, not sure if completionBlockWrapper is needed, since result nor error don't get changed.

Reply:
As we dicussed offline, we will keep current logic.